modules/social_features/social_profile/config/install/core.entity_form_display.profile.profile.default.yml

```diff
@@ -88,8 +88,7 @@ mode: default
 content:
   field_profile_address:
     weight: 7
-    settings:
```

Comment:
Please also right a hook_update_N or use the update helper module to create an update hook to remove these configurations from existing websites.

Reply:
Please ignore, as it is resolved by updated hook `function address_update_8104() {` https://www.drupal.org/files/issues/2019-04-04/2838457-30-default-value.patch